src/theme/ThemedComponents/AccountCard/AccountCard.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import { Card, CardHeader } from '@material-ui/core';
+import Typography from '@material-ui/core/Typography';
+
+import { ReactComponent as IDIcon } from '../../../components/identifier.svg';
+import EmailIcon from '@material-ui/icons/EmailOutlined';
+import PhoneIcon from '@material-ui/icons/PhoneOutlined';
+
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+	cardContainer: {
+		height: '152px',
+		width: '288px',
+	},
+	gridContainer: {
+		height: '100%',
+	},
+	cardHeader: {
+		textAlign: 'left',
+	},
+	leftStripe: {
+		color: 'white',
+		display: 'flex',
+		alignItems: 'center',
+		justifyContent: 'center',
+		margin: '0 auto',
+		height: '100%',
+		borderRadius: '8px 0px 0px 8px',
+	},
+	rightInfo: {
+		width: '100%',
+		height: '100%',
+		background: '#FFFFFF',
+		boxShadow: '0px 0px 8px rgba(196, 196, 196, 0.6)',
+		borderTopRightRadius: '8px',
+		borderBottomRightRadius: '8px',
+	},
+	accountInfo: {
+		display: 'flex',
+		alignItems: 'center',
+		justifyContent: 'flex-start',
+	},
+	iconStyles: {
+		display: 'flex',
+		alignItems: 'center',
+		justifyContent: 'center',
+	}
+});
+
+const AccountCard = ({ accountName, role }) => {
+	const classes = useStyles();
+
+	let stripeColor;
+
+	if (role === 'Parent') {
+		stripeColor = '#D74AC8';
```

Comment:
Change this to reflect how the color is determined with the stringToColor function